ios/RNTrackPlayer/RNTrackPlayer.swift

```diff
@@ -316,9 +319,21 @@ public class RNTrackPlayer: RCTEventEmitter, AudioSessionControllerDelegate {
                                           bookmarkOptions: options["bookmarkOptions"] as? [String: Any])
         }
 
+        if let interval = options["progressUpdateEventInterval"] as? NSNumber, Int(truncating: interval) > 0 {
```

Comment:
```suggestion
        if let interval = options["progressUpdateEventInterval"] as? NSNumber, interval.intValue > 0 {
```

I think this would work too..